libs/libc/stdio/lib_rdflush_unlocked.c

```diff
@@ -98,12 +94,10 @@ int lib_rdflush(FAR FILE *stream)
 
       if (fseek(stream, -rdoffset, SEEK_CUR) < 0)
```

Comment:
Do we need unlocked version of `fseek` or replace it with `_NX_SEEK` variant?

Reply:
spec doesn't define fseek_unlock

Reply:
Yes, I know it, but it is not logical to call API the is doing `flockfile(stream);` from `_unlocked` API variant, isn't it?

Reply:
Done, using lseek to do.